IntelliJ Project/src/main/java/com/globant/musicstore/dao/InvoiceDAO.java

```diff
@@ -1,9 +1,12 @@
 package com.globant.musicstore.dao;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 import com.globant.musicstore.entity.Invoice;
 
 public interface InvoiceDAO extends JpaRepository<Invoice, Long>{
+	@Query("SELECT invoice From Invoice invoice WHERE invoice.invoiceId IN :id")
+	Invoice getinvoice (Long id);
```

Comment:
Same here. this query is not necessary

Reply:
This query is needed IntInvoiceAlbum because a foreign key in the IntInvoiceAlbum